### PR TITLE
LPS-155316 Add test

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:forward

**Comment by liuxingye** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplTest.java [hunk 1]
```diff
@@ -62,6 +68,31 @@ public class PortalImplTest {
 	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Test
+	public void testGetCanonicalURLWithURLSeparatorInFriendlyURL()
```
Comment (AliciaGarciaGarcia) [inline modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplTest.java:72]: the test is failiing on CI ? can you review it?

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplTest.java (hunk 1 above)
Comment (adolfopa) [reply modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplTest.java:72]: Yes, I'm on it. I'll close this PR and send it later with fixes.

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1257401_3159

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by austinchiang** [comment]
ci:reevaluate:1377401_2746

**Comment by austinchiang** [comment]
The backend failure has been happening for a while https://testray.liferay.com/home/-/testray/case_results/1876954955/history?tab=result&hideFilterPin=true

**Comment by austinchiang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/118846

**Pull request closed without merging**